woocommerce/woocommerce-admin
# Add Payments Task Track Events

This PR builds upon/depends on #2960 and #2897.
See #2671.

It implements all of the `wcadmin_tasklist_payment_*`  track events listed in the events spreadsheet.

### Testing Directions:

* Either clear out the  `woocommerce_onboarding_payments` option, or complete running through the payments task so that you can restart from the beginning.
* * In your debug console, enter `localStorage.setItem( 'debug', 'wc-admin:*' )`.
* Walk through the payment method flows outlined in #2960 and #2897 to make sure the events in the spreadsheet are fired. `wcadmin_tasklist_payment_choose_method`, `wcadmin_tasklist_payment_install_method`, and `wcadmin_tasklist_payment_connect_method`.
